solitude/settings/products.py

```diff
@@ -0,0 +1,20 @@
+# A definiton of Products for Payments for Firefox Accounts. Marketplace
+# defines these dynamically. However for the moment we will define these from
+# configuration files.
+from decimal import Decimal
+
+products = {
```

Comment:
Wasn't sure how big or complicated this was going to get. I wanted something small and self contained that people could read and update without having to plow through settings. But then again it could just be a seperate file that we import.